themes/default/content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -2529,6 +2529,610 @@ Status: 200 OK
+| `scheduleCron`      | string | body  | cron expression for when to run drift detection                                                              |
+| `autoRemediate`     | bool   | body  | whether to remediate any detected drift                                                                      |
+
+#### Example
+
+```bash
+curl \
+  -H "Accept: application/vnd.pulumi+8" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+  --request POST \
+  --data '{"scheduleCron":"{scheduleCronValue}","autoRemediate":"{autoRemediateValue}"}' \
+  https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/deployments/drift/schedules
+```
+
+#### Default response
+
+```
+Status: 200 OK
+```
+
+```
+{
+  "id": "12345678-8102-447f-b246-e9ec85786e23",
+  "orgID": "87654321-8b3b-418a-8c01-5ddd0e00bace",
+  "scheduleCron": "0 0 * * *",
+  "nextExecution": "2024-04-20 00:00:00.000",
+  "paused": false,
+  "kind": "deployment",
+  "definition": {
+    "programID": "12345678-9f94-4a92-8bfb-c2da724fa65b",
+    "request": {
+        "inheritSettings": true,
+        "operation": "detect-drift",
+        "operationContext": {
+            "options": {
+                "remediateIfDriftDetected": true
+            }
+        }
+    }
+  },
+  "created": "2024-04-19 01:00:00.000",
+  "modified": "2024-04-19 01:00:00.000",
+  "lastExecuted": null
+}
+```
+
+### Create TTL Schedule
+
+```
+POST /api/stacks/{organization}/{project}/{stack}/deployments/ttl/schedules
+```
+
+#### Parameters
+
+| Parameter           | Type   | In    | Description                                                                                                  |
+|---------------------|--------|-------|--------------------------------------------------------------------------------------------------------------|
+| `organization`      | string | path  | organization name                                                                                            |
+| `project`           | string | path  | project name                                                                                                 |
+| `stack`             | string | path  | stack name                                                                                                   |
+| `timestamp`         | string | body  | ISO timestamp of when to destroy the stack                                                                   |
+| `deleteAfterDestroy`| bool   | body  | whether to delete the stack after resources are destroyed                                                    |
+
+#### Example
+
+```bash
+curl \
+  -H "Accept: application/vnd.pulumi+8" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+  --request POST \
+  --data '{"timestamp":"{timestampValue}","deleteAfterDestroy":"{deleteAfterDestroyValue}"}' \
```

Comment:
Make it a real example, so fill in the values instead of `{timestampValue}` and `{deleteAfterDestroyValue}`. Applies throughout.

Reply:
Throughout the doc, I saw examples of both and wasn't sure which one was best. Updating to real values.